db/seeds.rb

```diff
@@ -51,16 +55,18 @@ def seed_samples(project, sample_count)
       { name: "#{project.namespace.parent.name}/#{project.name} Sample #{i}",
         description: "This is a description for sample #{project.namespace.parent.name}/#{project.name} Sample #{i}." }
     ).execute
-    seed_attachments(sample) if i < 10
+    seed_attachments(sample) if @sample_attachment_count < @sample_attachment_limit
   end
 end
 
 def seed_attachments(sample)
+  Rails.logger.info "seeding attachments... #{@sample_attachment_count}/#{@sample_attachment_limit}"
   sequencing_files.each do |f|
     a = sample.attachments.build
     a.file.attach(io: Rails.root.join('test/fixtures/files', f).open, filename: f.to_s)
     a.save!
   end
+  @sample_attachment_count += 1
```

Comment:
This can be removed with all the changes above

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/310/commits/ec1705f3c953272d7404d7efb995d35d49d31cf3